packages/world/ts/getWorldAbi.ts

```diff
@@ -0,0 +1,39 @@
+import { Client, Abi, Address, getAddress, parseAbi } from "viem";
+import { worldDeployEvents } from "./common";
+import { getBlockNumber, getLogs } from "viem/actions";
+
+import { getSystems } from "./getSystems";
+import { functionSignatureToAbiItem } from "./functionSignatureToAbiItem";
+
+export async function getWorldAbi({
+  client,
+  worldAddress,
+}: {
+  readonly client: Client;
+  readonly worldAddress: Address;
```

Comment:
I think this could also just take in `fromBlock` and `toBlock` to pass down into `getSystems`, rather than fetching the deploy block in here.

Reply:
Good idea, accepting `fromBlock` and `toBlock` now.